src/content/docs/en/guides/deploy/deno.mdx

```diff
@@ -173,5 +173,16 @@ If your project is stored on GitHub, the [Deno Deploy website](https://dash.deno
 
 <ReadMore>Read more about [SSR in Astro](/en/guides/server-side-rendering/).</ReadMore>
 
+#### Deploy Astro static site with CLI deployment
+1. build the project so that the contents of the `dist` folder are up to date.
+2. In the terminal cd to the `./dist` folder and run the deploy command with the following parameters
+
+```sh
+deployctl deploy --project=<MY-DENO-PROJECT> --entrypoint=https://deno.land/std@0.211.0/http/file_server.ts
+```
+
+All the contents from the `./dist` folder are uploaded to the project and served by `file_server.ts`. 
+
+
```

Comment:
```suggestion
```
My suggestion is, if we're not entirely sure about this part, then we just remove the instructions and add them back at a later time when we are! 

We can add the earlier change with the link, though!